metadata/service/inmemory/service_proxy.go

```diff
@@ -64,10 +64,10 @@ func (m *MetadataServiceProxy) GetExportedURLs(serviceInterface string, group st
 		return []*common.URL{}, nil
 	}
 
-	urlStrs := res.Result().([]string)
+	urlStrs := *res.Result().(*[]interface{})
```

Comment:
pls add comment for this change

Reply:
@AlexStocks @LaurenceLiZhixin This code is outdated. I have keep it same as 3.0 branch.